packages/api/tests/lib/FileUtils/test-setS3FileSize.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const test = require('ava');
+const { setS3FileSize } = require('../../../lib/FileUtils');
+
+test('setS3FileSize() returns the value of the fileSize property in the size field', async (t) => {
+  const file = { fileSize: 1234 };
+
+  t.deepEqual(
+    await setS3FileSize(file),
+    { size: 1234 }
+  );
+});
+
+test('setS3FileSize() returns the value of the size property in the size field', async (t) => {
+  const file = { size: 1234 };
+
+  t.deepEqual(
+    await setS3FileSize(file),
+    file
+  );
+});
+
+test('setS3FileSize() prefers size over fileSize', async (t) => {
```

Comment:
I'm wondering if we have `size` and `fileSize`, should we remove `fileSize`? I'm debating because I don't really expect this case, but I'm wondering if leaving `fileSize` in could mask downstream errors where someone is using `fileSize` in error?

Reply:
Ideally, yes. I left it in for backwards compatibility, and since the `CnmToGranule` version used in our integration tests still creates an object with `fileSize`, but eventually I would like to see all references to `fileSize` deleted.